README.rst

```diff
@@ -15,26 +15,22 @@ Testing |image1|
 ----------------
 
 The source is tested using the python unittest framework. To execute all
-the unit tests, nose is utilized by the setup script (execute via `python setup.py test``. The tests are also
-executed by `Travis CI <https://travis-ci.org/Myoldmopar/py-idd-idf>`__ for each commit.
+the unit tests, simply run ``nosetests``. The tests are also
```

Comment:
Some cleanup here, will need to clean it up more I'm sure....and just make it md come on.